core/include/traccc/clusterization/detail/sparse_ccl.hpp

```diff
@@ -39,16 +48,22 @@ TRACCC_HOST_DEVICE inline unsigned int find_root(
 /// @param L an equivalance table
 ///
 /// @return the rleast common ancestor of the entries
-template <template <typename> class vector_t>
-TRACCC_HOST_DEVICE inline unsigned int make_union(vector_t<unsigned int>& L,
-                                                  const unsigned int& e1,
-                                                  const unsigned int& e2) {
+TRACCC_HOST_DEVICE inline unsigned int make_union(unsigned int* L,
+                                                  unsigned int e1,
+                                                  unsigned int e2,
+                                                  unsigned int size) {
```

Comment:
Same here, `L` should be a vector.

Reply:
Moved the code to using `vecmem::device_vector<unsigned int>`...